hack/operatorhub/catalog.go

```diff
@@ -0,0 +1,76 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
I think if the main package is split into multiple source files it cannot be run simply with `go run main.go` which is what is documented in the README and what the build automation will try to do.